### DISCUSSION TITLE
Update-2022-20-15

### DISCUSSION THREAD

**Discussion by kaveh808** [kaveh808/kons-9#137] [Announcements]
Implemented sample Signed Distance Functions and demo of visualization with isosurfaces.

The image below is using the function:

```
(min (sd-cut-hollow-sphere p 0.7 .3 .1)
     (sd-box-frame p (p! 0.8 0.3 0.8) .05)
     (sd-capped-torus p (p! .5 -.866 0.0) 0.7 0.1))
```

In file `demo-isosurface.lisp`

<img width="955" alt="Screen Shot 2022-10-15 at 10 46 24 AM" src="https://user-images.githubusercontent.com/103860793/196007303-9a3c871c-5be1-455b-aaf0-18c0dadfd3b1.png">

**Comment by lukego** [comment]
Holy shoes, that looks fantastic!
